lib/ansible/modules/windows/win_lineinfile.ps1

```diff
@@ -421,7 +421,7 @@ If ($state -eq "present") {
 	}
 
 	If ($insertbefore -and $insertafter) {
-		Add-Warning('Both insertbefore and insertafter parameters found, ignoring "insertafter=$insertafter"');
+		Add-Warning $result 'Both insertbefore and insertafter parameters found, ignoring "insertafter=$insertafter"'
```

Comment:
Happy to merge it now (apart from the quoting side), we can always revisit it as yes you are correct there seems to be different ways we are doing this stuff.